common/lib/dependabot/pull_request_creator/pr_name_prefixer.rb

```diff
@@ -307,6 +309,19 @@ def recent_azure_commit_messages
           map(&:strip)
       end
 
+      # FIXME: Should this paginate through all commits, or is 100 enough?
+      def recent_bitbucket_commit_messages
+        @recent_bitbucket_commit_messages ||=
+          bitbucket_client_for_source.commits(source.repo)
+
+        @recent_bitbucket_commit_messages.
+          reject { |c| bitbucket_commit_author_email(c) == dependabot_email }.
+          reject { |c| c.fetch("message")&.start_with?("Merge") }.
+          map { |c| c.fetch("message") }.
+          compact.
```

Comment:
```suggestion
          map { |c| c.fetch("message") }.
          compact.
          reject { |m| m.start_with?("Merge") }.
```

Idea: reduce to commits with a `message` before filtering on the message contents.

Reply:
Done, with a modification.